internal/webhook/external/path.go

```diff
@@ -6,9 +6,13 @@ import (
 	"strings"
 )
 
-func GenerateWebhookPath(project, provider, token string) string {
-	input := []byte(project + token)
+// GenerateWebhookPath generates a unique path for a webhook based on the
+// provided webhook receiver name, project name, kind, and token.
+// The path is formatted as "/webhook/{kind}/{hash}" where the hash is
+// a SHA-256 hash of the concatenated webhook receiver name, project name, and token.
+func GenerateWebhookPath(name, project, kind, token string) string {
```

Comment:
Suggest "secret" instead of "token" because it's more generic. Individual receiver types might pass a secret here that isn't a token, but is something else.